src/engine/sparqlExpressions/StringExpressions.cpp

```diff
@@ -399,6 +399,28 @@ class ConcatExpression : public detail::VariadicExpression {
 using EncodeForUriExpression =
     StringExpressionImpl<1, decltype(encodeForUriImpl)>;
 
+// TO_NUMERIC (int or double)
+auto InvalidArg = std::errc::invalid_argument;
+auto OutOfRange = std::errc::result_out_of_range;
```

Comment:
As you only use them inside the `toNumeric` function, you can also define them in there to not leak outside.